integration-tests/go.mod

```diff
@@ -33,8 +33,8 @@ require (
 	github.com/slack-go/slack v0.12.2
 	github.com/smartcontractkit/chain-selectors v1.0.21
 	github.com/smartcontractkit/chainlink-automation v1.0.4
-	github.com/smartcontractkit/chainlink-common v0.2.2-0.20240816204408-654165b6ee33
-	github.com/smartcontractkit/chainlink-testing-framework v1.34.5
```

Comment:
@DeividasK please use `chainlink-testing-framework v1.34.5`. I'm not sure about `chainlink-common v0.2.2-0.20240816204408-654165b6ee33`. Do you have any reasons to use a different version for it? Also, once you update the mods in `integration-tests/` then please do:
```
cd integration-tests/ 
go mod tidy
``
and
```
cd integration-tests/load
go mod tidy
```